spec/views/companies/index.html.erb_spec.rb

```diff
@@ -2,6 +2,28 @@
 
 require 'rails_helper'
 
-RSpec.describe 'companies/index.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'companies/index.html.erb rendering locals in a partial', type: :view do
+  context 'with 2 companies' do
+    let(:companies) do
+      Company.create!([
+        {
+          name: 'MERCATOR',
+          symbol: 'PLMRCTR00015',
+        },
+        {
+          name: 'MOSTALPLC',
+          symbol: 'PLMSTPL00018',
+        },
+      ])
+    end
```

Comment:
I would add en empty line after `end`:
```suggestion
    end
    
```